### PR TITLE
Validate special characters in notebook name

### LINKED ISSUE #1026: Unable to load notebooks with a colon in the name
If I create a notebook and include a colon in the name, I then cannot open the notebook.
The UI shows me:

```
Terra is preparing your notebook
- Waiting for notebook runtime to be ready
- Error loading notebook, retry number 5...
```
and the error bar at the top shows details:

```
Notebook cannot be launched
Unable to copy notebook to cluster, was it renamed or deleted in the Workspace Bucket?
```

When I try to set the name of a notebook, I get a warning:

```
NAME CAN'T CONTAIN ANY OF THESE CHARACTERS: "#[]*?"
```

So I assumed that colons are okay.

I was naming my notebooks as:

```
Py3: Getting Started
```

When I rename this to:

```
Py3 - Getting Started
```

the notebook loads fine.

### PR DESCRIPTION
**Pull request by katevoss**
Fixes #1026 

- Notebook name cannot contain , : ; @ $ % = + / \
- Increased font size and weight to make validation errors slightly bolder

### CODE REVIEW

File: src/components/input.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ const styles = {
   },
   validationError: {
     color: colors.red[0],
-    fontSize: 10, fontWeight: 500, textTransform: 'uppercase',
+    fontSize: 11, fontWeight: 600, textTransform: 'uppercase',
```
Comment (katevoss) [inline src/components/input.js:39]: Increased fontWeight to make the validation errors more visible

File: src/components/notebook-utils.js [hunk 2]
```diff
@@ -16,8 +16,8 @@ import validate from 'validate.js'
 export const notebookNameValidator = existing => ({
   presence: { allowEmpty: false },
   format: {
-    pattern: /^[^#[\]*?]*$/,
-    message: 'can\'t contain any of these characters: "#[]*?"'
+    pattern: /^[^#[\]*?:;@$%+=\\,/]*$/,
+    message: 'can\'t contain any of these characters: # [ ] * ? , : ; @ $ % + = / \\ '
```
Comment (katevoss) [inline src/components/notebook-utils.js]: Does anyone know how to add a newline to the message? `\n` and `\r` did not work.

Comment (breilly2) [reply src/components/notebook-utils.js]: `message` can be a component such as a `div()`with multiple children, including something like `br()` or whatever formatting/layout you want to use. When you use a component, it doesn't automatically prepend the field name so you'll have to add it to the text yourself.

**Review by elizzhao — APPROVED** [review]
Looks good to me!

**Review by panentheos — APPROVED** [review]
Looks good.

**Pull request merged**